### PR TITLE
Tag color fix for app expanded view

### PR DESCRIPTION
**Pull request by ibolton336**
- Show correct tag colors in app expanded view.
https://issues.redhat.com/browse/TACKLE-420

### CODE REVIEW

File: pkg/client/src/app/pages/applications/components/application-tags/application-tags.tsx [hunk 1]
```diff
@@ -44,38 +44,58 @@ export const ApplicationTags: React.FC<ApplicationTagsProps> = ({
...
-
-              // Tags
-              newTags.set(tagType.id!, [
-                ...(newTags.get(tagType.id!) || []),
-                e,
-              ]);
-            }
+          Promise.all(
+            tagValidResponses.map((tag) =>
+              getTagTypeById(tag?.tagType?.id || 0)
```
Comment (gildub) [inline pkg/client/src/app/pages/applications/components/application-tags/application-tags.tsx:62]: Why not use React queries ?

**Review by gildub — COMMENTED** [review]
It seems tags are still showing in gray only color.

File: pkg/client/src/app/pages/applications/components/application-tags/application-tags.tsx (hunk 1 above)
Comment (ibolton336) [reply pkg/client/src/app/pages/applications/components/application-tags/application-tags.tsx:62]: This is a bug fix for the existing code. Sure we can move to queries. But this PR aims to fix the bug with the current code.

**Comment by ibolton336** [comment]
> It seems tags are still showing in gray only color.
Maybe try to pull down the latest changes - Where are you seeing the tags with gray only color?
Screen Shot 2022-04-08 at 8 23 04 AM

File: pkg/client/src/app/pages/applications/components/application-tags/application-tags.tsx (hunk 1 above)
Comment (gildub) [reply pkg/client/src/app/pages/applications/components/application-tags/application-tags.tsx:62]: Ah yes of course it's already using legacy query, never mind.

**Review by gildub — APPROVED** [review]
Latest pull worked for me.

**Pull request merged**